documentation/projects.md

```diff
@@ -0,0 +1,114 @@
+# 🧪 Bolt projects
+
+🧪 **Experimental:** Bolt projects are experimental and may change in future
+versions of Bolt. Make sure you've updated to the most recent version of Bolt to
+take full advantage of this feature.
+
+One of the first things you'll do when you start using Bolt is create a Bolt
+project. A Bolt project is a simple directory that serves as the launching point
+for Bolt. You run Bolt from your project directory, and the directory houses
+your Bolt configuration files and your Bolt content, such as plans and tasks.
+
+In addition to working with your local Bolt content, Bolt projects give you a
+way to share that content with other users in your organization. You can create
+orchestration that is specific to the infrastructure you're working with, and
+then commit the project directory to git for others to consume.
+
+Here is an example of a typical project with a task, a plan, and an inventory file:
+
+```console
+.
+├── bolt-project.yaml
+├── inventory.yaml
+├── plans
+│   └── myplan.yaml
+└── tasks
+    ├── mytask.json
+    └── mytask.py
+```
+
+Bolt projects are experimental. Make sure you've [updated to the latest version
+of Bolt](./bolt_installing.md) to get the most out of this feature.
+
+## Creating a Bolt project
+
+To get started with a Bolt project:
+1. Create a `bolt-project.yaml` file in the root of your Bolt project directory.
+   This can be an existing directory, or a new one you make.
+2. Name your project by adding a `name` key to the top of `bolt-project.yaml`.
+   Project names can contain only lowercase letters, numbers, and underscores,
+   and begin with a lowercase letter. For example:
+   ```yaml
+   name: myproject
+   ```
+3. Develop your Bolt plans and tasks in `plans` and `tasks` directories in the
+   root of the project directory, next to `bolt-project.yaml`.
+
+Bolt treats a directory as a Bolt project as long as a `bolt-project.yaml` file
+exists at the root of the directory and contains a `name` key. Bolt loads tasks
+and plans from the `tasks` and `plans` directories and namespaces them to the
+project name.
+
+## Configuring a project
+
+The `bolt-project.yaml` file holds options to configure your Bolt project, as
+well as options to control how Bolt behaves when you run it from inside the
+project.
```

Comment:
```suggestion
well as options to control how Bolt behaves when you run the project.
```

Reply:
Run 'with' a project? I'm not sure what good phrasing is there.

Reply:
Hmmm. Is the expectation that user will not always run Bolt from inside their project directory? I guess I was making an assumption here that most users will - especially if they have more than one Bolt project. This makes me think that I should expand a little further on `--project <DIRECTORY_PATH>`.

Reply:
Ya, there isn't really an expectation that users will always run Bolt from inside the project directory. This is especially true for users who are only using the default project directory, as that will be loaded from anywhere.

Reply:
I don't think we should emphasize `--project` though. In general, you should be inside the project directory _except_ for the special case where you're using the default project. I think "run the project" works here.

Reply:
Okay, I'm going to go with "run a project" for now. Further up in the doc, I've said 
> "You run Bolt from your project directory, and the directory houses
> your Bolt configuration files and your Bolt content, such as plans and tasks." 

I think that sets the desired interaction for the user. There are more details for how Bolt loads projects, and more ways to configure the project location, in the Project directories doc, but we can always revisit if this causes confusion.